parser.go

```diff
@@ -467,6 +474,7 @@ func useTheUmbrella() {
 }
 
 func tauntTheEagle() {
+	eagleWatching = false
```

Comment:
should eagleWatching be false here? If the player comes back to the yard then the eagle would be gone after this one interaction. This is what I was trying to accomplish in an earlier iteration that used the eagle's discovered attribute, but it was decided that the eagle should always be discovered.

Reply:
The player can always look at the eagle when you enter the yard, even if you've been there before.